src/domain/app/header/navbar/navbar.module.scss

```diff
@@ -4,9 +4,6 @@
 @import 'mixins';
 @import 'variables';
 
-$logoHeight: 2.5rem;
```

Comment:
Why? you reuse it or something?

Reply:
I figured it could be nice to have those two variables together in the same place. A future dev might be confused as to why we have them in different files / have trouble finding them. 

Do you think I should move the height one back?